express/src/services/cognito/CognitoClient.ts

```diff
@@ -122,6 +124,16 @@ If the app is being deployed to PaaS then you may have to update manifest.yaml o
         return await this.cognitoClient.send(command);
     }
 
+    async forgotPassword(email: string): Promise<ForgotPasswordCommandOutput> {
+        console.debug(`sending forgotPassword code to this address : ${{email}}`);
```

Comment:
I don't think we want to log users' email addresses.

Reply:
Removed.